exporter/azure_monitor/common/common.go

```diff
@@ -52,3 +53,12 @@ type RemoteDependency struct {
 	Ver int				`json:"ver"`
 	Type string			`json:"type"`
 }
+
+type Request struct {
+	Name string 		`json:"name"`
+	Id string 			`json:"id"`
+	Duration string 	`json:"duration"`
+	ResponseCode string `json:"responsecode"`
+	Success bool 		`json:"success"`
+	Url interface{} 	`json:"url"`

```

Comment:
Why is Url an interface{} ?

Reply:
Yeah, it could be converted to a string before being stored and still work. Url is stored in sd.attributes which is part of Attributes in SpanData which is a map of interface{} (see https://godoc.org/go.opencensus.io/trace#SpanData).